package.json

```diff
@@ -5,6 +5,12 @@
     "url": "https://github.com/BrightspaceUILabs/facet-filter-sort.git"
   },
   "name": "@brightspace-ui-labs/facet-filter-sort",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "/components"
```

Comment:
Include `custom-elements.json` as well so the docs can pick that up.

Reply:
Should I delete the file and have it auto-generate before publish like `core` does?

Reply:
Oooh yes that'd be much better!